src/main/java/com/sucy/skill/listener/ExperienceListener.java

```diff
@@ -86,6 +91,17 @@ public void onCraft(CraftItemEvent event) {
         }
     }
 
+    @EventHandler(priority = EventPriority.LOWEST)
+    public void onExperienceGain(PlayerExperienceGainEvent event) {
+        Player player = event.getPlayerData().getPlayer();
+        if (event.isCancelled()
+                && event.getSource() == ExpSource.PLUGIN
+                && exempt.containsKey(player.getUniqueId())
+                && exempt.get(player.getUniqueId()) == event.getExp()) {
+            event.setCancelled(false);
+        }
+    }
```

Comment:
@neYaANK I wanted to set up a way to still let the class settings take effect, while still allowing the exp to go through, so this is what I came up with. I wasn't able to test it, though. So if you wouldn't mind making sure that this works as I expect it to, that would be awesome :D

Reply:
It does work, but it's worth mentioning that it requires `exp.yml`'s `enabled` field to be `true` which is by default `false` and that may cause some confusion

Reply:
Oh does it? It doesn't even fire off the event if it is false?